shared/src/main/scala/com/nitro/nmesos/singularity/model.scala

```diff
@@ -65,7 +65,10 @@ object model {
     customExecutorCmd: Option[String] = None,
     env: Map[String, String] = Map.empty,
     healthcheckUri: Option[String] = None,
-    healthcheckPortIndex: Option[Int] = None
+    healthcheckPortIndex: Option[Int] = None,
```

Comment:
The singularity model looks good, but to make nmesos understand it we need to add the yaml config model and conversion too.
 see [Yaml SingularityConf](https://github.com/Nitro/nmesos/blob/master/shared/src/main/scala/com/nitro/nmesos/config/model.scala#L59) and the conversion between [Yaml and Json](https://github.com/Nitro/nmesos/blob/master/shared/src/main/scala/com/nitro/nmesos/singularity/ModelConversions.scala#L126)